src/nerve_tools/configure_nerve.py

```diff
@@ -66,13 +66,16 @@ def generate_subconfiguration(
     zk_topology_dir,
     zk_location_type,
     zk_cluster_type,
+    location_depth_mapping,
```

Comment:
I don't think you need this. Available location's are guaranteed to be most specific to least.